OpenEdXMobile/src/main/java/org/edx/mobile/view/RegisterActivity.java

```diff
@@ -80,7 +80,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         setTitle(R.string.register_title);
 
-        environment.getAnalyticsRegistry().trackScreenView(Analytics.Screens.LAUNCH_ACTIVITY);
+        environment.getAnalyticsRegistry().trackScreenView(Analytics.Screens.REGISTER);
```

Comment:
This needs to be documented though I guess (and maybe a ticket is needed for iOS).